### ISSUE TITLE
Unable to delete Split DNS Nameserver entry if more than 1 are present

### ISSUE THREAD

**Issue by Gerion42** [tale/headplane#40]
Trying to delete the Split DNS entry `split.example.com` when there are more than 1 Split DNS entry present. The following error message is present in the log: 
```
Error: Expected YAML collection at split. Remaining path: split.example.com
    at YAMLMap.deleteIn (/app/node_modules/.pnpm/yaml@2.4.5/node_modules/yaml/dist/nodes/Collection.js:89:19)
    at YAMLMap.deleteIn (/app/node_modules/.pnpm/yaml@2.4.5/node_modules/yaml/dist/nodes/Collection.js:87:25)
    at YAMLMap.deleteIn (/app/node_modules/.pnpm/yaml@2.4.5/node_modules/yaml/dist/nodes/Collection.js:87:25)
    at Document.deleteIn (/app/node_modules/.pnpm/yaml@2.4.5/node_modules/yaml/dist/doc/Document.js:175:29)
    at patchConfig (file:///app/build/server/index.js?t=1728630686000:709:18)
    at action$2 (file:///app/build/server/index.js?t=1728630686000:6768:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.callRouteAction (/app/node_modules/.pnpm/@remix-run+server-runtime@2.10.2_typescript@5.5.3/node_modules/@remix-run/server-runtime/dist/data.js:37:16)
    at /app/node_modules/.pnpm/@remix-run+router@1.17.1/node_modules/@remix-run/router/router.ts:4817:21
    at callLoaderOrAction (/app/node_modules/.pnpm/@remix-run+router@1.17.1/node_modules/@remix-run/router/router.ts:4879:16)
```

**Comment by Gerion42** [comment]
A similar error appears when trying to remove an ip for a split dns section

**Comment by tale** [comment]
Able to reproduce, will investigate ASAP.

**Comment by tale** [comment]
Fixed in `0.3.4`
